build.gradle

```diff
@@ -131,13 +131,27 @@ afterEvaluate {
 
     buildRpm {
         arch = 'NOARCH'
-        archiveName "${packageName}-${version}.rpm"
         dependsOn 'assemble'
+        finalizedBy 'renameRpm'
```

Comment:
i thought you already added this in another pr

Reply:
I think `fetch` and `rebase` are needed to make the dev branch having the latest code.

Reply:
Oh that is good to know for next time. Wasn't sure why it was showing up again.